acados/ocp_nlp/ocp_nlp_cost_ls.c

```diff
@@ -607,22 +661,51 @@ void ocp_nlp_cost_ls_update_qp_matrices(void *config_, void *dims_,
 
     int nx = dims->nx;
     int nu = dims->nu;
+    int nz = dims->nz;
     int ny = dims->ny;
     int ns = dims->ns;
 
     // initialize hessian of lagrangian with hessian of cost
     blasfeo_dgecp(nu + nx, nu + nx, &memory->hess, 0, 0, memory->RSQrq, 0, 0);
 
+    // copy Cyt into Cyt_tilde
+    blasfeo_dgecp(nu + nx, ny, &model->Cyt, 0, 0, &work->Cyt_tilde, 0, 0);
+    // copy y_ref into y_ref_tilde
+    blasfeo_dveccp(ny, &model->y_ref, 0, &work->y_ref_tilde, 0);
+    if (nz > 0) { // eliminate algebraic variables and update Cyt and y_ref
+        // swap dzdu and dzdx
```

Comment:
Put in the correct format `dzdux_tran` directly when getting the sensitivities from the integrator's `S_algebraic`: there is no use of `dzdxu`, right?

Reply:
right, addressed in 940fc22.